recipes/newrelic/infrastructure/darwin.yml

```diff
@@ -63,8 +64,30 @@ install:
         - |
           IS_BREW_INSTALLED=$(which brew | wc -l)
           if [ $IS_BREW_INSTALLED -eq 0 ] ; then
-            echo "brew is required to run the newrelic install. Please install brew and re-run the installation." >&2
-            exit 16
+            while :; do
+              echo -n "brew is required to run the newrelic install. Do you want to install it? Y/N (default: Y) "
+              read answer
+              echo ""
+              NEW_RELIC_CONTINUE=$(echo "${answer^^}" | cut -c1-1)
+              if [[ -z "$NEW_RELIC_CONTINUE" ]]; then
+                NEW_RELIC_CONTINUE="Y"
+              fi
+              if [[ "$NEW_RELIC_CONTINUE" == "N" ]]; then
+                echo "brew is required to run the newrelic install. Please install brew and re-run the installation." >&2
+                exit 131
+              fi
+              if [[ "$NEW_RELIC_CONTINUE" == "Y" ]]; then
+                break
+              fi
+              echo -e "Please type Y or N only."
+            done
+            echo -e "{{.ARROW}}Installing brew...{{.GRAY}}"
+            echo ""
+            BREW_INSTALL_SUCCESS=$(sudo -i -u $SUDO_USER /bin/bash -c 'curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh | bash > /dev/null 2>&1; echo $?')
```

Comment:
Looks like it works, but part of the brew installation requires manual input from the user to install xcode components and accept the license agreement, so the install script fails before this finishes. Is there a way that we can somehow wait until `which brew` returns an exit code 0 instead?